src/content/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign.mdx

```diff
@@ -52,11 +52,9 @@ To use automated user management to import users from your identity provider:
 3. If you use Azure AD, Okta, or OneLogin, use the applicable guide: [Azure AD](/docs/azure-scimsso-application-configuration) \| [Okta](/docs/okta-scimsso-application-configuration) \| [OneLogin](/docs/onelogin-scimsso-application-configuration). 
 4. If you **don't** use one of the above services, you'll need to:
   * In the authentication domain UI, [enable SCIM as the source of users](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#source-users). 
-  * Use our [SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management) to integrate with your identity provider service. 
-  * [Configure SAML SSO](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#saml).
-  * If you're on [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), you'll likely want to downgrade some of your billable [full users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type) to free [basic users](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#user-type). To do this, use the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#add-users).
-  * Once your users are provisioned in New Relic, you'll need to assign access grants to give them access to accounts. [Learn more about access grants.](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts) 
-6. **Highly recommended**: Set a time zone in your identity provider. How you do this will vary by identity provider. If not set in your identity provider, our UI shows UTC time zone dates/times. Time zone is specified in IANA Time Zone database format, also known as the "Olson" time zone database format (e.g., "America/Los_Angeles").
+  * Use our [SCIM API](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management) to integrate with your identity provider service. See [the SCIM API tutorial](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim) for all the steps involved. 
+5. Optional: Whether you use one of the dedicated apps (Azure AD, Okta, OneLogin) or not, you have the option to manage user type via our SCIM API. For more, see [Manage user type](/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim#manage-user-type). 
+6. **Highly recommended**: Set a time zone for your users in your identity provider. How you do this will vary by identity provider. If not set in your identity provider, our UI shows UTC time zone dates/times. Time zone is specified in IANA Time Zone database format, also known as the "Olson" time zone database format (e.g., "America/Los_Angeles").
```

Comment:
Do we really need to provide this level of detail about time zones?

This detail seems to be implying possible incompatibility issues with time zones. If that's so, I'd rather get real explicit about that as a possible problem and information on how to mitigate that. (Does knowing it's an "Olson" time zone help with that, for example?)

Reply:
it's a real problem apparently so they wanted to emphasize that it's important to set up, and it'll vary a lot by identity provider type.